spring-kafka-docs/src/main/antora/modules/ROOT/pages/streams.adoc

```diff
@@ -82,7 +82,7 @@ A new `KafkaStreams` is created on each `start()`.
 You might also consider using different `StreamsBuilderFactoryBean` instances, if you would like to control the lifecycles for `KStream` instances separately.
 
 You also can specify `KafkaStreams.StateListener`, `Thread.UncaughtExceptionHandler`, and `StateRestoreListener` options on the `StreamsBuilderFactoryBean`, which are delegated to the internal `KafkaStreams` instance.
-Also, apart from setting those options indirectly on `StreamsBuilderFactoryBean`, starting with _version 2.1.5_, you can use a `KafkaStreamsCustomizer` callback interface to configure an inner `KafkaStreams` instance.
+Also, apart from setting those options indirectly on `StreamsBuilderFactoryBean`, starting with _version 2.1.5_, you can use a `KafkaStreamsCustomizer` callback interface to configure an inner `KafkaStreams` instance and (from _version 3.3.0_) instantiate a custom implementation of `KafkaStreams`.
```

Comment:
Maybe split them into two lines?

Reply:
No splitting of sentences to multi lines, please:  https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Reply:
For ease of readability thoughts on this structure?
![image](https://github.com/user-attachments/assets/2c6a9797-665b-45d9-8b43-e29147c7eff8)

Reply:
Well, I would go different sentences at all.

Reply:
OK, I kept sentences to one line where it made sense but separated some paragraphs so that it it is easier to read (I had actually struggled to read the initial flow as one paragraph has many different concepts mixed together). Here's the new look:

![image](https://github.com/user-attachments/assets/ef4aef91-aa70-47ba-8fdc-5096523bf840)

Since I'm not 100% sure what the standard is here, feel free to suggest edits or push to this branch directly.